catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/security/ShareableMetacardPolicyExtension.java

```diff
@@ -125,14 +134,26 @@ private KeyValueCollectionPermission isPermitted(
 
     // filter out all implied permissions
     Function<Set<String>, KeyValueCollectionPermission> filterPermissions =
-        (implied) -> {
-          List<KeyValuePermission> values =
-              permissions
+        (impliedKeys) -> {
+          KeyValueCollectionPermission implied =
+              new KeyValueCollectionPermission(
+                  match.getAction(),
+                  impliedKeys
+                      .stream()
+                      .map(key -> new KeyValuePermission(key, grouped.get(key)))
+                      .collect(Collectors.toList()));
+
+          List<Permission> values =
+              allPerms
```

Comment:
Left a comment on this on your other PR, don't forget to change it here too.

Reply:
They are not dependent so I'm removing the change from this PR.